l10n_ec_delivery_note/models/account_edi_document.py

```diff
@@ -0,0 +1,303 @@
+import logging
+from os import path
+
+from psycopg2 import OperationalError
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+
+EDI_DATE_FORMAT = "%d/%m/%Y"
+DEFAULT_BLOCKING_LEVEL = "error"
+
+_logger = logging.getLogger(__name__)
+
+
+class AccountEdiDocument(models.Model):
+    _inherit = "account.edi.document"
+
+    l10n_ec_delivery_note_id = fields.Many2one(
+        comodel_name="l10n_ec.delivery.note",
+        string="Delivery delivery_note",
+        required=False,
+        ondelete="cascade",
+    )
+    move_id = fields.Many2one(required=False)
```

Comment:
This can be a mess with another implementations? Maybe you should use a constrain or another way to handle required on another cases except yours

Reply:
can you give a example please, i not understand?